README.md

```diff
@@ -150,3 +150,13 @@ $ keytool -list -v -keystore "$HOME/.android/debug.keystore" \
     https://support.google.com/firebase/answer/7015592
 
 This includes the API key and URL for your new Firebase project.
+
+### Set up google-cloud-sdk
```

Comment:
Do developers only need to set this up if they want to build with Google Cloud Build? If so it may be worth changing the heading to "Building with Google Cloud Build (optional)".

Reply:
Docker image only needs to be updated whenever there's a change in the `Dockerfile`, otherwise not. Thanks for the suggestion! Updated.